spec/features/consumer/shopping/embedded_groups_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+
+feature "Using embedded shopfront functionality", js: true do
+
+  Capybara.server_port = 9999
+
+  describe 'embedded groups' do
+    let(:enterprise) { create(:distributor_enterprise) }
+    let!(:group) { create(:enterprise_group, enterprises: [enterprise], permalink: 'group1', on_front_page: true) }
+
+    before do
+      Spree::Config[:enable_embedded_shopfronts] = true
+      Spree::Config[:embedded_shopfronts_whitelist] = 'localhost'
+
+      page.driver.browser.js_errors = false
+      Capybara.current_session.driver.visit('spec/support/views/group_iframe_test.html')
+    end
+
+    after do
+      Spree::Config[:enable_embedded_shopfronts] = false
+    end
```

Comment:
RSpec's `around` is better suited than pairs of `before` and `after`. I'd use it here. See https://relishapp.com/rspec/rspec-core/v/3-7/docs/hooks/around-hooks. What do you think?